### ISSUE TITLE
documentation on how to use the new forecast and anomaly SQL functions

### ISSUE THREAD

**Issue by wgong** [taosdata/TDengine#28200]
I have read this blog [一条SQL，带你进入AI时代](https://mp.weixin.qq.com/s?__biz=MzIzNzg5MTcxNA==&mid=2247485195&idx=1&sn=58e0ad69b0e81462679b67568f8a4dd0&chksm=e9a3165aba10a3d786b7ef0608f9d15edcbc49db73067bcf039fde2ac91312e10eaeb24b39f3&mpshare=1&srcid=0801Q5hoXvcDgW8onnQcV8uw&sharer_shareinfo=09d603f86dea4467fcb60f79988765bb&sharer_shareinfo_first=9453b7bc95c51669decc72997a0c8f7f&from=singlemessage&scene=1&subscene=10000&clicktime=1727575085&enterid=1727575085&sessionid=0&ascene=1&fasttmpl_type=0&fasttmpl_fullversion=7402152-en_US-zip&fasttmpl_flag=0&realreporttime=1727575085484#rd)  and try it out at cloud.tdengine.com

after launching a DB instance in AWS cloud (region=us-east-1), 
The TDengine version is `server version: 3.3.2.11.202409060825`.

Running this SQL 
```
select _rowts, forecast(ts, voltage, 20) from test.meters;
```
I got `Error: Func not exists`, Is it still under development?

Thanks

**Comment by yu285** [comment]
this is not involved in the TDengine Cloud  yet , if you are interested ,we can arrange a communication , Do you have weChat ? or other contacts?

**Comment by wgong** [comment]
Yes, I am interested, here is my wechat![Screenshot_20241025_235907_WeChat.jpg](https://github.com/user-attachments/assets/382fd459-4219-43d6-b480-cb22c7eeaa85)

**Comment by yu285** [comment]
added
